scripts/storage-emulator-integration/tests.ts

```diff
@@ -1309,6 +1309,29 @@ describe("Storage emulator", () => {
             expect(md.downloadTokens.split(",")).to.not.deep.equal([token]);
           });
       });
+
+      it("#uploadResumable", async () => {
```

Comment:
nit: please include some information in the name as to what you're testing, eg uploadResumableDoesNotRequireMultipleAuthHeaders

Reply:
Done.